Tools/scripts/deepfreeze.py

```diff
@@ -13,11 +13,11 @@
 import time
 import types
 from typing import Dict, FrozenSet, TextIO, Tuple
-
```

Comment:
Please restore this blank line (conventionally I like a blank line between stdlib imports and local ones).

Reply:
Okay I think it was removed by vscode isort.

Reply:
Fixed